### PR TITLE
Adds citation functionality to sechuds

### PR DESCRIPTION
**Pull request by Dingo-Dongler**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/49017 allowing sechuds to give people citations.

## Why It's Good For The Game

Fine the assistant for breaking windows instantly!

## Changelog
:cl:
add: Examining people with sechuds will let you give someone an on the spot citation. Pick up that trash!
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
> ## About The Pull Request
> Ports [tgstation/tgstation#49017](https://github.com/tgstation/tgstation/pull/49017) allowing sechuds to give people citations.
> 
> ## Why It's Good For The Game
> Fine the assistant for breaking windows instantly!
> 
> ## Changelog
> 🆑
> add: Examining people with sechuds will let you give someone an on the spot citation. Pick up that trash!
> /🆑

https://github.com/BeeStation/BeeStation-Hornet/pull/1790

File: config/game_options.txt [hunk 1]
```diff
@@ -558,6 +558,9 @@ ROUNDSTART_TRAITS
 ## Sets shift time to server time at roundstart. Overridden by RANDOMIZE_SHIFT_TIME ##
 #SHIFT_TIME_REALTIME
 
+## Maximum fine for a citation
+MAXFINE 2000
```
Comment (ike709) [inline config/game_options.txt:562]: You need to add this to `config/Sage/game_options.txt` as well.

**Comment by ike709** [comment]
@zeskorion What even is your comment?

**Comment by zeskorion** [comment]
> @zeskorion What even is your comment?

pointing to an apparently merged pr that ports the same feature

**Comment by Dingo-Dongler** [comment]
No this is specifically for sechuds, not citations overall.

**Comment by ike709** [comment]
@zeskorion You can't read.

**Review by ike709 — APPROVED** [review]

**Comment by ike709** [comment]
@Crossedfall Config changes.

**Comment by Ohagi-Chan** [comment]
@Dingo-Dongler but doesn't the other PR say it adds citations to sechuds on the very first line?

**Comment by ike709** [comment]
Why does nobody know how to read? 

No @Ohagi-Chan

**Comment by Mick1299** [comment]
Zeskorion's quote quotes this pr, not the pr they reference at the end.

That pr added citations, this pr allows you to give citations via sechud

**Review by Crossedfall — APPROVED** [review]
Config changes applied.

**Pull request merged**